app/views/tutorials/_form.html.erb

```diff
@@ -23,6 +23,6 @@
     <%= f.input :content, label: false, input_html: { class: "summernote-plugin" } %>
   </div>
   <div class="actions">
-    <%= f.button :submit, value: "Submit Changes for Review", class: "primary-button" %>
+    <%= f.submit, value: "Submit Changes for Review", class: "primary-button" %>
```

Comment:
LGTM and also tested. i pull the branch and tried it on my computer. creates the tutorial just fine. i think this can be merged in.